lib/puppet/provider/service/systemd.rb

```diff
@@ -26,6 +26,7 @@
   defaultfor :operatingsystem => :ubuntu
   notdefaultfor :operatingsystem => :ubuntu, :operatingsystemmajrelease => ["10.04", "12.04", "14.04", "14.10"] # These are using upstart
   defaultfor :operatingsystem => :cumuluslinux, :operatingsystemmajrelease => ["3", "4"]
+  defaultfor :operatingsystem => :gentoo
```

Comment:
Is this really correct? I know I had to apply it because I selected systemd but OpenRC is still the default init system.

Reply:
Ah, good point.